locksmith/src/utils/key.ts

```diff
@@ -0,0 +1,64 @@
+import { ethers } from 'ethers'
```

Comment:
Hum, don't do a utils there, let's use an operations file please to remain consistent!